### PR TITLE
Make updates per product owner suggestions

### PR DESCRIPTION
**Pull request by ledbutter**
Jira [reference](https://jira.concur.com/browse/BUD-2325)

### CODE REVIEW

File: src/api-reference/budget/getting-started.md [hunk 1]
```diff
@@ -52,16 +52,6 @@ Name|Description|Endpoint
...
-This API uses this optional scope:
-
-Name|Description|Endpoint
----|---|---
-`LIST` |GET List Items, necessary for budget tracking value `listKey` if using list fields for budget tracking. |POST Budget Item
-
-## <a name="dependencies"></a>Dependencies
-
-SAP Concur clients must purchase Budget in order to use this API. The developer may use the following SAP Concur API to get optional information:
-* List Item v3.0, to retrieve the budget tracking value `listKey`
```
Comment (jonikamelcher) [inline src/api-reference/budget/getting-started.md:64]: @ledbutter Hello. Please keep the Dependencies section with the text: "SAP Concur clients must purchase Budget in order to use this API." Thanks.

**Review by jonikamelcher — CHANGES_REQUESTED** [review]

File: src/api-reference/budget/getting-started.md (hunk 1 above)
Comment (ledbutter) [reply src/api-reference/budget/getting-started.md:64]: Done, budget is going out of pre-release, is this the place to remove the `{% include prerelease.html %}` reference from [Budget.markdown](https://github.com/concur/developer.concur.com/blob/preview/src/api-explorer/v4-0/Budget.markdown) or is that done automatically as part of the migration to being in GA?

Comment (jonikamelcher) [reply src/api-reference/budget/getting-started.md:64]: Thank you! We will be removing the prerelease reference as part of the migration a little later.

**Review by jonikamelcher — APPROVED** [review]

**Comment by jonikamelcher** [comment]
:shipit:

**Pull request merged**